client/src/ts/dd_main.ts

```diff
@@ -7,11 +7,14 @@ import canDefs = require('./can_msg_defs');
 
 // SOC
 
-function mapSocPercent(voltage: number): number {
-  // Use a poor linear approximation for now based on:
-  // https://lygte-info.dk/info/BatteryChargePercent%20UK.html
+function mapSocPercent(v: number): number {
+  // Based on a discharge curve fit of polynomial degree 6.
   // TODO(ckitagawa): Make this better if time allows.
-  const result = 3.15 * voltage - 373;
+  const max_capcity = 4368.3804;  // Ah
```

Comment:
s/capcity/capacity/

4368Ah? Our pack should be ~126Ah.

Reply:
All the curves stop at 122 but thanks for the catch. I accidentally multiplied by 36.